integration_test/cleanup/cleanup-dr-buckets.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ex
+
+cd integration_test
+
+#initialize terraform
+terraform init -input=false
+#destroy using terraform
+echo "Destroying DR S3 buckets in Disaster Recovery account"
+terraform destroy \
+  -auto-approve \
+  -input=false
```

Comment:
The folder integration_test contains the tf files, and the state bucket points to specific resources. Setup is done in the shared/setup-dr-buckets.sh to avoid copy/pasting the backend setup.

Reply:
Errors in TF do not presently halt script execution, so I do not feel that it is safe to delete the TF-specific resources.

Reply:
sounds good. I missed the other scripts you were running in the first place